### PR TITLE
Upload artifacts to GCS from verify test - part 1

### PR DESCRIPTION
**Pull request by csrwng**
Changes to the base image script and release-ci tool

### CODE REVIEW

**Comment by csrwng** [comment]
@stevekuznetsov ptal

File: tools/release-ci/pkg/config/presubmit.go [hunk 1]
```diff
@@ -25,12 +25,15 @@ type Presubmit struct {
...
 // store their under the directory of their PR
 func (p *Presubmit) GcsPath() string {
-	return fmt.Sprintf("%s/%s%s/%d/%s/%d/", PullRequestPrefix, PullRequestIndex, repositoryPath(p.RepoOwner, p.RepoName), p.PullNumber, p.JobName, p.BuildNumber)
+	return fmt.Sprintf("%s/%s%s/%d/%s/%d%s/", PullRequestPrefix, PullRequestIndex, repositoryPath(p.RepoOwner, p.RepoName), p.PullNumber, p.JobName, p.BuildNumber, testPath(p.TestName))
 }
 
 // Aliases are the paths to symlink to the GcsPath.
 // Pre-submit jobs alias under the pull directory.
 func (p *Presubmit) Aliases() []string {
+	if len(p.TestName) > 0 {
```
Comment (stevekuznetsov) [inline tools/release-ci/pkg/config/presubmit.go]: We've messed up separation of concerns here

Comment (csrwng) [reply tools/release-ci/pkg/config/presubmit.go]: Will fix

**Comment by csrwng** [comment]
Removed test name from prow config and made it an argument to the sidecar command.

File: cluster/ci/origin/images/release-base/log-pods.sh [hunk 2]
```diff
@@ -48,7 +49,11 @@ while [[ "${finished}" == "no" ]]; do
 		fi
 		totalCount=$(( totalCount + 1 ))
 		completed_pods+=("${podname}")
-		oc log "${podname}"
+		if [[ -n "${container_name}" ]]; then
```
Comment (stevekuznetsov) [inline cluster/ci/origin/images/release-base/log-pods.sh]: You have to check `-n` here but you just assign to `${2}` -- will the assignment fail when there are not 2+ args?

Comment (csrwng) [reply cluster/ci/origin/images/release-base/log-pods.sh]: you're right, it will fail... let me add a default

**Comment by stevekuznetsov** [comment]
Also did we redeploy Jenkins with the updated plugin to allow us to run the pipeline build that we need to run after this goes in?

**Pull request merged**